tests/cli/workflow/taxprofiler/test_cli_taxprofiler_compound_commands.py

```diff
@@ -49,3 +49,46 @@ def test_taxprofiler_start(
 
     # THEN command should not include resume flag
     assert "-resume" not in caplog.text
+
+
+def test_taxprofiler_start_available(
```

Comment:
can we move rnafusion and taxprofiler star-available to general parametrised tests as part of nf-analysis? they are quite redundant.

I'm thinking something like what was done for `test_config_case_without_options`

for example:
```
@pytest.mark.parametrize(
    "context",
    ["rnafusion_context", "taxprofiler_context"],
)
def test_start_available(cli_runner: CliRunner, context: CGConfig, request):
```

Reply:
and the same for the other tests you included

Reply:
I will move the `start_available `to general parametrised tests and leave the rest after fixture-refactoring by @talnor.